app/templates/events/view/tickets.hbs

```diff
@@ -1,17 +1,15 @@
-<div class="ui grid">
```

Comment:
@anu-007 The `ui grid` was removed removed from the parent in #134. If you want a child view to use a certain class please add it inside the child view. Do not change this file it will be taken care of in #121.